EIPS/eip-3220.md

```diff
@@ -0,0 +1,123 @@
+---
+eip: 3220
+title: Crosschain Identifier Specification
+author: Weijia Zhang (@weijia31415), Peter Robinson (@drinkcoffee)
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-10-21
+---
+
+## Simple Summary
+
+Today chainid definition as specified in EIP-155 is an integer type.  This is not unique enough to identify all public and private blockchains.  A blockchain identifier (crosschain id) should be unique and satisfy the following requirements:
+
+should provide identification, description, and discovery of blockchains.
```

Comment:
The simple summary should generally just be 1-2 sentences. I recommend moving this to motivation.

Reply:
Modified and move this content to Motivation